util/circular-chk.py

```diff
@@ -0,0 +1,51 @@
+import glob
+import re
+from sys import exit, argv
+from pathlib import Path
+
+strict = False
+if len(argv) > 1 and argv[1] == '--strict' :
+    strict = True
+
+regexExclude = r"keymaps\/(.*)via\/"
+reg = re.compile(regexExclude)
+filesToscan = Path('keyboards').glob('**/rules.mk')
+illegalSRC = list()
+
+for f in filesToscan:
+    p = str(f).replace('\\','/')
+    basepath = p.replace('keyboards/','')
+    basepath_arr = basepath.split('/')
+
+    # Skip 'via' file
+    if reg.search(p):
+        continue
+
+    # Read file as lines
+    lineList = list()
+    with f.open() as file:
+        try:
+            lineList = file.readlines()
+        except:
+            # sometimes encoding issues with japanese characters for example
+            pass
+
+    # Concat multilines
+    lineList = '\n'.join(lineList).replace('\\\n','').split('\n')
+    for l in lineList:
+        if not l.startswith('SRC ='):
```

Comment:
What about files where `SRC` has been indented?

Reply:
Isn't clang taking care of this?

Reply:
Clang only formats C source code, not make files.

Reply:
So, is that a formatting makefile issue or a script parsing issue here?

Reply:
Whitespace is valid in the makefile, so I would lean towards script parsing issue.

Probably something simple like `line = line.strip()` will solve it.